source/building-templates.md

```diff
@@ -59,8 +59,8 @@ default.
 
 <p class="tip"> Tip: the default template set uses includes to insert the
 header, footer and such, but you're free to use
-<a href="http://twig.sensiolabs.org/doc/templates.html#template-inheritance">Template Inheritance</a>
-if you prefer. </p>
+[Template Inheritance](http://twig.sensiolabs.org/doc/templates.html#template-inheritance)
```

Comment:
Does this work? I thought our markdown parser ignores markdown inside html tags?